static/js/classroom/game.js

```
@@ -0,0 +1,566 @@
+  // Classroom assets
+  this.load.image('tiles', '/static/images/classroom/classroom_tiles.png');
+  this.load.image('whiteboard', '/static/images/classroom/whiteboard.png');
+  this.load.image('chair', '/static/images/classroom/chair.png');
+  this.load.image('chair-taken', '/static/images/classroom/chair-taken.png');
+  this.load.image('desk', '/static/images/classroom/desk.png');
+  this.load.image('window', '/static/images/classroom/window.png');
+  this.load.image('door', '/static/images/classroom/door.png');
+  this.load.image('interaction_icon', '/static/images/classroom/interaction_icon.png');
+
+  // Add progress bar
+  let progressBar = this.add.graphics();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'progressBar' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)